### DISCUSSION TITLE
The antenna of the ESP32 on the AI Deck has fallen off, what should I do？

### DISCUSSION THREAD

**Discussion by wyj-bit** [bitcraze/discussions#1144] [Q&A] [answered]
Today while refreshing the AI Deck, I accidentally knocked off the antenna of the ESP32. Is it possible to reconnect it through soldering, or are there any other methods for repair? 
![89b8e8859fa74c49962ee65da749940](https://github.com/bitcraze/discussions/assets/151802680/877f1592-5d19-4786-a021-a9d68244e2be)
I also have another questions. I only did the 'flash wifi' and it got stuck at 99%. I performed this on a Windows system, including updating Crazyflie and AIdeck firmware. Do I need to perform 'Flash Wifi' on a Linux system? Regarding the issue of getting stuck at 99%, do I need to use a JTAG adapter for debugging?

**Comment by tobbeanton** [comment]
The wifi part will work very poorly without an antenna. Can you see if the pads are ripped off or not? Especially the 3 pads close together as those are the feeding pad and GNDs. Trying to solder the antenna back is one option which requires the 3 pads to still be there. You can use hot-glue after soldering it, to secure it in place.
This [discussion](https://github.com/orgs/bitcraze/discussions/916) might also help a bit.

Yes you will need a JTAG debugger.

**Reply by wyj-bit** [reply]
Thank you for your advice. However, after the antenna fell off, it is still possible to perform the flash.Could you please take a look at these pictures and let me know if soldering is possible?
![dedf70523c1c2c51f082a8a98e489de](https://github.com/bitcraze/discussions/assets/151802680/874a1072-5c23-4dc5-a4eb-f5a1e437b85f)
![61453872949efc27e8e59df0e843c77](https://github.com/bitcraze/discussions/assets/151802680/b473ddf5-5a08-466a-9d48-69ffb06073d8)
![081076b1d829d0071bc35eeba798e67](https://github.com/bitcraze/discussions/assets/151802680/9770cc5c-28d1-41ac-bf6b-5982fc78e0f8)

**Reply by tobbeanton** [reply, marked as answer]
Yes, the important pads looks intact (it is the 3 pads together to the right in the picture). The single pad to the left is ripped off but not really used so use hot-glue to keep it in place and to make it more sturdy.

Similar to this but I suggest filling it with even more hot-glue once you verified the antenna is working again.
![image](https://github.com/bitcraze/discussions/assets/6593186/69f292ee-587b-4945-a45e-96365b200cd3)

**Reply by wyj-bit** [reply]
I understand, and I sincerely appreciate your valuable advice.
